src/app/core/core.module.ts

```diff
@@ -4,14 +4,15 @@ import { NavbarComponent } from './components/navbar/navbar.component';
 import { FooterComponent } from './components/footer/footer.component';
 import { SearchModule } from './../shared/components/search/search.module';
 import { AddButtonModule } from './../shared/components/add-button/add-button.module';
+import { MatIconModule } from '@angular/material/icon'
 
 @NgModule({
   declarations: [
     HeaderComponent,
     NavbarComponent,
     FooterComponent
   ],
-  imports: [SearchModule, AddButtonModule],
+  imports: [SearchModule, AddButtonModule, MatIconModule],
```

Comment:
MatIconModule should be imported in the place where it used directly. HeaderModule in our case. Please follow the same structure for components as we have on SharedModule.